src/raftstore/store/fsm/peer.rs

```diff
@@ -2380,6 +2413,7 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         }
         self.fsm.peer.size_diff_hint = 0;
         self.fsm.peer.compaction_declined_bytes = 0;
+        self.register_split_region_check_tick();
```

Comment:
seems it is unnecessary

Reply:
Split regions may still need to be split again.

Reply:
it will register tick in `on_ready_split_region` to split again, so need here

Reply:
Oh, forget about what I have said. Here is just to make it robust, as the task may be lost for some reason.